src/quote/getQuoteAndBuyCoverInputs.ts

```diff
@@ -94,6 +100,8 @@ async function getQuoteAndBuyCoverInputs(
   slippage: number = DEFAULT_SLIPPAGE / SLIPPAGE_DENOMINATOR,
   ipfsCidOrContent: string | IPFSContentForProductType[ProductTypes] = '',
   nexusApiUrl = 'https://api.nexusmutual.io/v2',
+  commissionRatio?: number,
+  commissionDestination?: Address,
```

Comment:
Here I was thinking we should switch to an object param instead. We have 10 params for this method already.